backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -123,13 +123,19 @@ def generate_test_dict():
         "failReasons": defaultdict(int),
         "configs": defaultdict(lambda: defaultdict(int)),
         "issues": {},
+        "failedWithUnkownIssues": 0,
     }
 
 
 def is_record_in_tree_head(record, tree):
     return record["build__checkout__git_commit_hash"] == tree["headGitCommitHash"]
 
 
+def update_unknown_issues(tests_or_boots, status):
+    if status == "FAIL":
+        tests_or_boots["failedWithUnknownIssues"] += 1
```

Comment:
the key `failedWithUnknownIssues` isn't the same that you put in generate test dict `(failedWithUnkownIssues)` - line 126

Reply:
Typo. Already changed it in the generate_test_dict function.